sdk/communication/azure-communication-network-traversal/CHANGELOG.md

```diff
@@ -0,0 +1,17 @@
+# Release History
+
+## 1package-2021-06-21-preview
+
+The first preview of the Azure Communication Relay Client has the following features:
+
+- get a Relay Configuration by creating a CommunicationRelayClient
+
+### Added
+- Added CommunicationRelayClient in preview.
+- Added CommunicationRelayClient.get_relay_configuration in preview.
+
+<!-- LINKS -->
+[read_me]: https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/communication/
+[documentation]: https://docs.microsoft.com/azure/communication-services/quickstarts/access-tokens?pivots=programming-language-python
```

Comment:
This is the documentation for `identity`, if there's no `networktraversal` documentation remove this